docs/cedarling/cedarling-policy-store.md

```diff
@@ -14,101 +14,192 @@ The Cedarling Policy Store uses a JSON file named `cedarling_store.json` to stor
-- **cedar_policies** : (*Object*) Object containing one or more policy IDs as keys, with their corresponding objects as values. See: [cedar_policies schema](#cedar-policies-schema).
-- **cedar_schema** : (*String*) The Cedar schema, encoded in Base64 format.
-- **trusted_issuers** : (*Array of [TrustedIssuer](#trusted-issuer-schema)*) List of metadata for Trusted Issuers.
+- **policies** : (*Object*) Object containing one or more policy IDs as keys, with their corresponding objects as values. See: [policies schema](#cedar-policies-schema).
+- **schema** : (*String* | *Object*) The Cedar Schema. See [schema](#schema) below.
+- **identity_source** : (*Object of {unique_id => IdentitySource}(#trusted-issuer-schema)*) List of metadata for Identity Sources.
+
+### `schema`
+Either *String* or *Object*, where *Object* is preferred.
+
+Where *Object* - An object with `encoding`, `content_type` and `body` keys. For example:
+
+``` json
+"schema": {
+    "encoding": "none", // can be one of "none" or "base64"
+    "content_type": "cedar", // can be one of "cedar" or "cedar-json"
+    "body": "namespace Jans {\ntype Url = {"host": String, "path": String, "protocol": String};..."
+}
+```
+  Where *String* - The schema in cedar-json format, encoded as Base64. For example:
+``` json
+"schema": "cGVybWl0KAogICAgc..."
+```
 
 ## Cedar Policies Schema
 
-The `cedar_policies` field outlines the Cedar policies that will be used in Cedarling. Multiple policies can be defined, with policy requiring a unique `policy_id`.
+The `policies` field describes the Cedar policies that will be used in Cedarling. Multiple policies can be defined, with each policy requiring a `unique_policy_id`.
 
 ```json
-  "cedar_policies": {
+  "policies": {
     "unique_policy_id": {
+      "cedar_version" : "v2.4.7",
+      "name": "Policy for Unique Id",
       "description": "simple policy example",
       "creation_date": "2024-09-20T17:22:39.996050",
-      "policy_content": "cGVybWl0KAogICAgc..."
+      "policy_content": { ... },
     },
     ...
   }
 ```
 
 - **unique_policy_id**: (*String*) A uniqe policy ID used to for tracking and auditing purposes.
+- **name** : (*String*) A name for the policy
 - **description** : (*String*) A brief description of cedar policy
 - **creation_date** :  (*String*) Policy creating date in `YYYY-MM-DDTHH:MM:SS.ssssss`
-- **policy_content** : (*String*) The policy content, encoded in Base64 format.
+- **policy_content** : (*String* | *Object*) The Cedar Policy. See [policy_content](#policy_content) below.
+
+### `policy_content`
+Either *String* or *Object*, where *Object* is preferred.
+
+Where *Object* - An object with `encoding`, `content_type` and `body` keys. For example:
+
+``` json
+"policy_content": {
+    "encoding": "none", // can be one of "none" or "base64"
```

Comment:
There are some recent changes in the policy store schema (see https://github.com/JanssenProject/jans/wiki/Cedarling-Nativity-Plan#cedarling-policy-store). I think we need to sync with those changes.

Reply:
Already up to date with those changes. There is no canonical description that I can find of the json schema, so I've worked off output from Policy Designer, and from discussions that I know about. Please point out any discrepancies you find.

Note that the above change you've highlighted is exactly what enables human-readable test fixtures.

It is not required that Agama Lab Policy Designer must emit json using `encoding`, `content_type`, `body` fields. It can simply continue using a base64 encoded string exactly as it is now.